packages/store/package.json

```diff
@@ -76,7 +76,7 @@
     "@types/mocha": "^9.1.1",
     "@types/node": "^18.15.11",
     "ds-test": "https://github.com/dapphub/ds-test.git#e282159d5170298eb2455a6c05280ab5a73a4ef0",
-    "ejs": "^3.1.8",
+    "ejs": "^3.1.10",
```

Comment:
I don't think this is used in store, world, and world-modules? wonder if we can safely remove